qcfractal/interface/collections/collection.py

```diff
@@ -266,7 +266,10 @@ def _add_molecules_by_dict(client, molecules):
             flat_map_keys.append(k)
             flat_map_mols.append(v)
 
-        mol_ret = client.add_molecules(flat_map_mols)
```

Comment:
Perhaps we should have a loop like this implemented somewhere in FractalClient.

Reply:
I like some sort of wrapper:

```python
def chunk_list(data_list, chunk_size, tqdm=None):
    if tqdm is None:
        tqdm = is_jupyter()

   for x in range(0, len(data_list), chunk_size):
        yield data_list[x : x + chunk_size]

    # handle tqdm

```